docs/Manual/03 Schedules.md

```diff
@@ -274,9 +276,9 @@ schedule.pad(i, size)
 The `pad` transformation pads the beginning of dimension `i` with empty elements. This operation is meaningless by itself, but can be useful when used with splitting or fusing.
 
 ## Order-invariant schedules and safety
-We say that a schedule is *order-invariant* if its underlying logic doesn't depend on the execution order of its iterations. For example, schedules created from a single `Nest` (via a call to `create_schedule()`) are order-invariant. All of the schedules discussed so far have been order-invariant.
+A schedule is *order-invariant* if its underlying logic doesn't depend on the execution order of its iterations. For example, schedules created from a single `Nest` (via `create_schedule()` call) are order-invariant. All of the schedules discussed so far have been order-invariant.
 
-We say that a schedule is *safe* if its underlying logic is guaranteed not to change, regardless of how we transform it. Not all schedules are safe, but order-invariant schedules are. This is because all of the transformations introduced in this section merely change the order in which iterations are executed, without adding or removing any work.
+A schedule is *safe* if its underlying logic is guaranteed to remain intact regardless of the applied transformations. Not all schedules are safe, but order-invariant schedules are. This is because the transformations introduced in this section only change the execution order of iterations without adding or removing any work.
```

Comment:
I agree with your suggestion of removing "We say that..." here (it doesn't add much here)